tests/integration/element__lazy_search_test.py

```diff
@@ -50,11 +59,22 @@ def test_search_is_updated_on_next_actual_action_like_questioning_displayed(
     page = GivenPage(session_browser.driver)
     page.opened_empty()
 
-    element = session_browser.element('#will-be-existing-element-id')
-    page.load_body('<h1 id="will-be-existing-element-id">Hello kitty:*</h1>')
+    element = session_browser.element(
+        '#will-be-existing-element-id')
+    page.load_body(
+        '''
+        <h1 id="will-be-existing-element-id">Hello kitty:
+        *</h1>
+        '''
+    )
+
     assert element().is_displayed() is True
 
     page.load_body(
-        '<h1 id="will-be-existing-element-id" style="display:none">Hello kitty:*</h1>'
+        '''
+        <h1 id="will-be-existing-element-id" style="display:none">Hello kitty:
+        *</h1>
```

Comment:
should be broken with a common sense, not just at some point...

good version is more like this:
```
<h1 id="will-be-existing-element-id" style="display:none">
    Hello kitty:*
</h1>
```